src/obstacle_avoidance/include/MyNode.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Created By: Chris Heathe
+ * Created On:  October28, 2017
+ * Description: A node that will theoretically interpret lidar data,
+ *              and help Olaf navigate through a hole in a row of cones
+ *              Stress on *theoretically*
+ */
+
+#ifndef obstacle_avoidance_MYNODE_H
+#define obstacle_avoidance_MYNODE_H
+
+#include <iostream>
+#include <std_msgs/String.h>
+#include <sensor_msgs/LaserScan.h>
+#include <ros/ros.h>
+#include <sb_utils.h>
+#include <geometry_msgs/Twist.h>
+
+class MyClass {
```

Comment:
Similarly to the filename, we should rename this to something like `ObstacleAvoidanceNode`. CLion should make this pretty easy to do, just `right click on "MyClass" -> Refactor -> Rename` (I think)